src/Cdn/Cdn/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 1.3.0
```

Comment:
@hytao you can remove this `## Version 1.3.0` header and put the below line under `## Upcoming Release`